WebCryptoAPI/derive_bits_keys/derived_bits_length_vectors.js

```diff
@@ -9,25 +9,33 @@ var algorithms = {
         privateKey: {format: "raw", data: rawKey},
         deriveAlg: {name: "HKDF", salt: salt, hash: "SHA-256", info: info},
         derivation: new Uint8Array([49, 183, 214, 133, 48, 168, 99, 231, 23, 192, 129, 202, 105, 23, 182, 134, 80, 179, 221, 154, 41, 243, 6, 6, 226, 202, 209, 153, 190, 193, 77, 19]),
+        derivation384: new Uint8Array([49, 183, 214, 133, 48, 168, 99, 231, 23, 192, 129, 202, 105, 23, 182, 134, 80, 179, 221, 154, 41, 243, 6, 6, 226, 202, 209, 153, 190, 193, 77, 19, 165, 50, 181, 8, 254, 59, 122, 199, 25, 224,146, 248, 105, 105, 75, 84]),
+        derivation230: new Uint8Array([]),
```

Comment:
Bit nitpicky but perhaps we should make this one `undefined` as well (and same for the others)? Otherwise it may look a bit confusing here, like we're expecting an empty Uint8Array for 230 bits or so.
```suggestion
        derivation230: undefined,
```
(or we could just remove it altogether?)

Reply:
Good suggestion, thanks.